asm/macros/event.inc

```diff
@@ -1786,6 +1786,17 @@
 	waitfanfare
 	.endm
 
+	@ Updates the complex quest's variable to give it its next state.
+	.macro updatequest quest:req, questvar:req
+	addvar \questvar, 1
+	questmenu QUEST_MENU_BUFFER_QUEST_NAME \quest
+	bufferstring 1 gText_QuestUpdate
+	playse SE_PIN
+	msgbox gText_QuestUpdate, MSGBOX_SIGN
+	waitse
+	openquestmenu
```

Comment:
Why do we have to open the quest menu after it updates? This should either be a seperate macro OR take a TRUE / FALSE parameter.